src/hotspot/share/runtime/mutex.hpp

```diff
@@ -50,7 +50,8 @@ class Mutex : public CHeapObj<mtSynchronizer> {
        tty            = stackwatermark +   3,
        special        = tty            +   3,
        oopstorage     = special        +   3,
-       leaf           = oopstorage     +  10,
+       nosafepoint    = oopstorage     +   6,
+       leaf           = nosafepoint    +   6,
        safepoint      = leaf           +  10,
```

Comment:
It is somewhat confusing to have safepoint as an explicit rank now that all ranks above nosafepoint imply safepoint-ing.

Reply:
I thought there was still a lock that used this rank but there isn't, so I'll remove it.